Buildings/Fluid/HeatExchangers/WetCoilEffectivenessNTU.mo

```diff
@@ -233,7 +231,7 @@ protected
   flo flowRegime(fixed=false, start=flowRegime_nominal)
     "Heat exchanger flow regime";
 
-  Buildings.HeatTransfer.Sources.PrescribedHeatFlow preHea
+  Modelica.Thermal.HeatTransfer.Sources.PrescribedHeatFlow preHea
```

Comment:
The BuildingsPy merge script automatically replaces `Modelica.Thermal.HeatTransfer.Sources.PrescribedHeatFlow` with `Buildings.HeatTransfer.Sources.PrescribedHeatFlow`, so this is fine.

Reply:
I don't understand. The branch `IBPSASync_issue1109_wetCoilEpsNTU` results from running the BuildingsPy merge script, right? But the instance of `Modelica.Thermal.HeatTransfer.Sources.PrescribedHeatFlow` in `IBPSA/Fluid/HeatExchangers/WetCoilEffectivenessNTU.mo` was not replaced by `Buildings.HeatTransfer.Sources.PrescribedHeatFlow` since we see it here as a change from the master branch.